doc/alertmanager-mixin/dashboards/overview.libsonnet

```diff
@@ -0,0 +1,310 @@
+local grafana = import 'github.com/grafana/grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+
+{
+  grafanaDashboards+:: {
+    local clusterTemplate =
+      template.new(
+        name='cluster',
+        datasource='$datasource',
+        query='label_values(alertmanager_alerts, %s)' % $._config.clusterLabel,
+        current='',
+        hide=if $._config.showMultiCluster then '' else '2',
+        refresh=2,
+        includeAll=false,
+        sort=1
+      ),
+
+    'alertmanager-overview.json':
+
+      local alerts =
+        graphPanel.new(
+          'Alerts',
+          datasource='$datasource',
+          span=6,
+          format='none',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('alertmanager_alerts{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}' % $._config, legendFormat='{{ state }}'));
+
+      local alertsRate =
+        graphPanel.new(
+          'Alerts receive rate',
+          datasource='$datasource',
+          span=6,
+          format='ops',
+          stack=false,
+          fill=1,
+          legend_show=false,
+        )
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_received_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Received'))
+        .addTarget(prometheus.target('sum(rate(alertmanager_alerts_invalid_total{%(alertmanagerSelector)s, %(clusterLabel)s="$cluster"}[5m]))' % $._config, legendFormat='Invalid'));
```

Comment:
Since a Prometheus server sends each alert to each Alertmanager instance, this sum will increase proportionally by the number of instances in the cluster. For example, if you add a 4th instance to your formerly three-instance AM cluster, this number will suddenly go up by 33%. That might or might not be what you want.

On the other hand, it points to a problem if some alerts only reach some of the AM instances.

Just brainstorming here, but since the number of instances in an AM cluster is usually low (three to five or something), perhaps don't sum but show the graph for each instance? In that case, it might be better to have a separate panel for the invalid alerts (which are, hopefully, tiny anyway and might make more sense in a separate panel anyway).